src/VisualStudio/CSharp/Test/EventHookup/EventHookupTestState.cs

```diff
@@ -26,7 +25,8 @@ public EventHookupTestState(XElement workspaceElement, IDictionary<OptionKey, ob
             : base(workspaceElement, GetExtraParts(), false)
         {
             _commandHandler = new EventHookupCommandHandler(Workspace.GetService<IInlineRenameService>(),
-                Workspace.ExportProvider.GetExportedValue<IAsynchronousOperationListenerProvider>());
+                Workspace.ExportProvider.GetExportedValue<IAsynchronousOperationListenerProvider>(),
+                new EventHookupSessionManager(Workspace.GetService<IToolTipService>()));
```

Comment:
Now that it's MEF, you can probably just do Workspace.ExportProvider.GetExportedValue<EventHookupSessionManager>(). @sharwell is now on a (very good) crusade to avoid these surprises.

Reply:
I got this exception after changing it to Workspace.ExportProvider.GetExportedValue<EventHookupSessionManager>()

Microsoft.VisualStudio.Composition.CompositionFailedException : Expected 1 export(s) with contract name "Microsoft.CodeAnalysis.Editor.CSharp.EventHookup.EventHookupSessionManager" but found 0 after applying applicable constraints.

Reply:
@JieCarolHu You probably have to add it to the GetExtraParts() function below so it's included in the composition.

Reply:
thanks. fixed.